include/beman/inplace_vector/inplace_vector.hpp

```diff
@@ -99,12 +111,12 @@ struct inplace_vector_base : public inplace_vector_destruct_base<T, Capacity> {
   inplace_vector_base(const inplace_vector_base &other) noexcept(
       std::is_nothrow_copy_constructible_v<T>)
       : inplace_vector_destruct_base<T, Capacity>(other.size) {
-    std::ranges::copy(other.begin(), other.end(), begin());
+    std::copy(other.begin(), other.end(), begin());
```

Comment:
Maybe consider a calling delegate for `std::copy` that switches between `std::copy` and `std::ranges::copy` based on if C++20 is available? There may still be some advantages to using the `ranges` version when available.

Reply:
I don't think there's any difference between `std::copy` and `std::ranges::copy` in our use case?

See std::ranges::copy in [cppreference (variant 3)](https://en.cppreference.com/w/cpp/algorithm/ranges/copy).
See std::copy in [cpprefernce (variant 1)](https://en.cppreference.com/w/cpp/algorithm/copy)

Reply:
I don't have a strong opinion, was just a suggestion as there are (probably) small implementation differences between `std::copy` and `std::ranges::copy`; but I don't know that for sure. I think using `std::copy` everywhere is fine.

Reply:
Alright! Let's keep it std::copy, I can add a patch if needed in the future